geoportailv3/static/js/draw/featurepopupservice.js

```diff
@@ -112,4 +119,51 @@ app.FeaturePopup.prototype.getAnchor = function(feature) {
 };
 
 
+/**
+ * @param {ol.geom.Polygon} polygon
+ * @return {string}
+ */
+app.FeaturePopup.prototype.formatArea = function(polygon) {
+  var sourceProj = this.map_.getView().getProjection();
+  var geom = /** @type {ol.geom.Polygon} */ (polygon.clone().transform(
+      sourceProj, 'EPSG:4326'));
+  var coordinates = geom.getLinearRing(0).getCoordinates();
+  var area = Math.abs(ol.sphere.WGS84.geodesicArea(coordinates));
+  var output;
+  if (area > 1000000) {
+    output = parseFloat((area / 1000000).toPrecision(3)) +
+        ' ' + 'km<sup>2</sup>';
+  } else {
+    output = parseFloat(area.toPrecision(3)) + ' ' + 'm<sup>2</sup>';
+  }
+  return output;
+};
+
```

Comment:
this is a copy-paste from ngeo, as the methods are private.
@pgiraud @elemoine could this be made public so that we do not have to write the code twice?